### ISSUE TITLE
SP API needs to be separated from Seller Central account

### ISSUE THREAD

**Issue by moltar** [amzn/selling-partner-api-docs#581]
It was a terrible idea to require a Seller Central account for MWS, and unfortunately the same mistake was repeated for Selling Partner API.

It is way too easy to trigger an account violation *without doing anything or even selling* and the account gets shut down.

For example, having two accounts is a policy violation.

But at the same time SP API and MWS policies require separation of access. And how can we separate production and development access with one account? It is not possible. If I need developers to be able to login to Amazon Seller Central account and to create test and development applications.

> **Access Management.** Developers must assign a unique ID to each person with computer access to Amazon Information. **__Developers must not create or use generic, shared, or default login credentials or user accounts.__** Developers must implement baselining mechanisms to ensure that at all times only the required user accounts access Amazon Information. Developers must review the list of people and services with access to Amazon Information on a regular basis (at least quarterly), and remove accounts that no longer require access. Developers must restrict developer employees from storing Amazon data on personal devices. Developers will maintain and enforce "account lockout" by detecting anomalous usage patterns and log-in attempts, and disabling accounts with access to Amazon Information as needed.

If you cannot separate this out at this point, at least create some guidance for internal staff. Because after explaining many times that **I AM NOT A SELLER**, my answers are completely disregarded and they keep asking me to provide reinstatement plans or various other documentation, like utility bills.

None of this makes sense.

**Comment by sparkweb** [comment]
Completely agree with all of this. We've had our account shut down before simply because we aren't selling anything. At a certain point it just looks like a money-grab forcing developers to pay for a selling account where they aren't selling anything. And not just one account, but one in _each region_.

**Comment by charliecode** [comment]
@moltar This is a seriously valid point. I like how you took the requirements from what Amazon expects of devs while handling Selling Partners data. It shows a real dissonance between what is expected of us and what Amazon expects of itself in this regard. The two don't currently match up. In order to do any developing all config must be setup by signing into the root Seller Central account, this too makes no sense as a Selling Partner would need to give their developer access to their root Seller Central account, very insecure from a security standpoint and totally unnecessary. That goes completely against what is said here "Developers must implement baselining mechanisms to ensure that at all times only the required user accounts access Amazon Information." 

@sparkweb I also agree 100% that a paid Seller Central account absolutely should not be required. No other APIs require it, why does Amazon Marketplace require it? Probability is Amazon simply hasn't prioritized it yet, I know for a fact they know about this and have even shown an interest in correcting it. But I also think it's only fair that Amazon realize the optics of this simply don't look good. Not only from a greed standpoint, but more importantly from a fair market standpoint. $39.99 might not seem like a lot of money to most engineers but there are a TON of SP-API devs who reside in countries where their wages are far less than most and the $39.99 monthly fee could easily take them out of the game. That simply isn't fair and it most certainly isn't right.

This is a nice discussion, I appreciate the respectful manner in which the previous comments were made, it's obvious the intent is to make things more secure and better for all involved. I'm confident Amazon will hear these concerns and fix them appropriately, sooner than later seems most appropriate considering the optics.

**Comment by tomhirschfeld** [comment]
+1, we are running into an issue right now where we want to integrate with vendors but cannot because we do not have a vendor central account, only seller central account, and vendor central accounts are invite only. Very strange, would be great for one of the maintainers here to help comment on the roadmap here

**Comment by sparkweb** [comment]
@tomhirschfeld we had to wait for a vendor central customer who would add us as a user on their account before even starting the development work. We finally got on and applied for an application and got denied because they didn't like our PII answers, despite the fact that we have an approved Seller Central app. We have to reapply--but it seems very wrong that our developer app on the vendor central side will be tied to a client account and if the client removed our access I suspect that we would lose access to the app. This seems so backwards and I can't imagine that it's _really_ how this is supposed to work.

**Comment by userbabu** [comment]
Yes, Amazon should listen to our needs and separate developer accounts from seller accounts.

**Comment by jimmy-ross-xapix** [comment]
> We've had our account shut down before simply because we aren't selling anything.

do they shut down the account even if you are paying the monthly fee? just trying to understand the risks.

**Comment by jimmy-ross-xapix** [comment]
wow, even if you have created an application that is hosted on the marketplace and is being used by many other Seller accounts?

**Comment by moltar** [comment]
> wow, even if you have created an application that is hosted on the marketplace and is being used by many other Seller accounts?

Yup!

And even if you have an old (aged) account.

And even if you had no history of any other violations in the past 5 years.

And there is no recourse or any human to talk to. You can open case logs, but you will be met with canned replies that make no sense, because they will be addressed to you, as a seller, which you aren't.

Your best bet is to contact one of the Amazon lawyers. The quote I got was $3,500 for the case to reinstate the account, and that has no guarantee, just "best effort".

**Comment by charliecode** [comment]
@moltar @jimmy-ross-xapix My last client was able to create a separate Selling on Amazon account which was only used for development (no selling anything at all) and it never got shut down. Amazon was contacted prior to creating it and they allowed it. I developed on that account for close to a year and it was never shut down. I can't guarantee it will work for everyone but it did work for us, keeping Amazon in the loop about what the account is used for is probably the key. Having said that, this is a pain point Amazon knows about and is actively working on features to help address, as @seanevan stated in #665. I would encourage you to chime in on that issue if you have any suggestions. Contrary to what some may feel, Amazon does listen to and help with these things, I can personally verify that was my case.

**Comment by charliecode** [comment]
@moltar I hear you, I would def encourage you to keep the dialogue open here and on #665 as it's obvious Amazon is keeping close ties on these comments from what Sean stated yesterday. Much better than blowing $3,500, you might be shooting yourself in the foot mentioning lawyers as it may tie the teams hands in helping further, just my two cents.

**Comment by moltar** [comment]
@seanevan I have come across this screen today, while applying for a SC account in the AU marketplace.

> We are making changes to simplify the developer experience. You will no longer be able to access the Login with Amazon Application Console on Seller Central, or make changes to your existing applications from the Application Console. To view or modify existing applications, or create a new application, sign in to Amazon Developer using your Seller Central login credentials. Your existing applications will be unaffected by this change and will be available on Amazon Developer once you sign in. Learn More. If you have any questions, please contact us.

![image](https://user-images.githubusercontent.com/491247/116960905-f8f3d500-accb-11eb-9913-126cd94e97e6.png)

Could you please confirm or deny that this is about the separation of SC from SP API?

Is this a global change?

Do we need to create one application for each region in the Amazon Developer console?

Thanks!

**Comment by moltar** [comment]
Never mind, it was for the "Login with Amazon" marketplace, which for some reason still shows up on the list of marketplaces, even though it isn't a marketplace.

![screenshot-20210504T134751-apfB766K](https://user-images.githubusercontent.com/491247/116968825-7412b680-acdf-11eb-8e6e-100875e0bfd6.png)

**Comment by cpeker-extenship** [comment]
Totally agree.

**Comment by moltar** [comment]
Not stale.

[GitHub stale bot considered harmful](https://drewdevault.com/2021/10/26/stalebot.html)

**Comment by jathos** [comment]
Is there any update on this?  

In April of 2021, on #665 Sean Evans says "We're actively working on planned features to reduce developers' reliance on seller accounts."  

As far as I can tell, two years later developers are still completely reliant on seller accounts and will still find themselves locked out of SP-API usage if their developer/seller account is suddenly disabled due to being a developer.

**Comment by zhihui** [comment]
Totally agree.

**Comment by lukewarlow** [comment]
This isn't stale. It's still a big issue.
